src/v2/Components/Authentication/Validators.tsx

```diff
@@ -4,9 +4,9 @@ export const email = Yup.string()
   .email("Please enter a valid email.")
   .required("Please enter a valid email.")
 
-const name = Yup.string().required("Name is required.")
+export const name = Yup.string().required("Name is required.")
 
-const password = Yup.string()
+export const password = Yup.string()
```

Comment:
What is `Yup`?

Reply:
It's a little library that let's you express a validation schema: https://github.com/jquense/yup

It plugs into Formik so that when inputs don't match the schema they get passed down as errors associated with the fields.